### PR TITLE
Update crd and add verify

### PR DESCRIPTION
**Pull request by qiujian16**
Follow up of pr #4

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,6 +21,12 @@ copy-crd:
 
 update-all: copy-crd update
 
+verify-crds:
+	$(deps_diff) {vendor/github.com/open-cluster-management/api/cluster/v1,manifests/hub}/0000_00_clusters.open-cluster-management.io_spokeclusters.crd.yaml || ( echo 'crd content is incorrect' && false )
```
Comment (deads2k) [inline Makefile]: we need to check all the yamls, right?

File: pkg/operators/hub/controller.go [hunk 2]
```diff
@@ -93,6 +97,10 @@ func (n *nucleusHubController) sync(ctx context.Context, controllerContext facto
 	klog.V(4).Infof("Reconciling HubCore %q", hubCoreName)
 
 	hubCore, err := n.nucleusLister.Get(hubCoreName)
+	if errors.IsNotFound(err) {
+		// HubCoree not found, could have been deleted, do nothing.
```
Comment (deads2k) [inline pkg/operators/hub/controller.go]: HubCore

File: Makefile (hunk 1 above)
Comment (qiujian16) [reply Makefile]: yes, changes to use script and check all yamls on the dir

File: go.mod [hunk 3]
```diff
@@ -4,7 +4,7 @@ go 1.13
 
 require (
 	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
-	github.com/open-cluster-management/api v0.0.0-20200506150956-355c9d6ef16b
+	github.com/open-cluster-management/api v0.0.0-20200512021938-5f0d5c12ea67
```
Comment (pmorie) [inline go.mod:7]: I wonder if we should start tagging releases soon, I have an allergic reaction to go mod pseudoversions

**Pull request merged**